src/mapgen/mg_schematic.cpp

```diff
@@ -137,8 +137,7 @@ void Schematic::blitToVManip(MMVManip *vm, v3s16 p, Rotation rot, bool force_pla
 		for (s16 z = 0; z != sz; z++) {
 			u32 i = z * i_step_z + y * ystride + i_start;
 			for (s16 x = 0; x != sx; x++, i += i_step_x) {
-				u32 vi = vm->m_area.index(p.X + x, y_map, p.Z + z);
-				if (!vm->m_area.contains(vi))
+				if (!vm->m_area.contains(v3s16(p.X + x, y_map, p.Z + z)))
```

Comment:
better: allocate this v3s16 before the condition and use the attributes in the vm->m_area.index() call directly. This permits to prevent double calculation

Reply:
Ok.

Reply:
Done.